private/src/gst/gstavbvideosrc.cpp

```diff
@@ -97,6 +106,11 @@ gst_avbvideosrc_class_init(GstAvbVideoSrcClass * klass)
             g_param_spec_string("stream-name", "Stream name",
                 "Name of the stream", DEFAULT_STREAM_NAME,
                 (GParamFlags)(G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS)));
+
+    g_object_class_install_property(gobject_class, PROP_STREAM_TYPE,
+            g_param_spec_uint("stream-type", "Stream type",
+                "Type of the stream", 1, 10, DEFAULT_STREAM_TYPE,
```

Comment:
Why between 1 and 10 are valid, if only 1 and 2 really are?

Reply:
Yea, STREAM_TYPE_MAX is now set to 2.